       1:      1  1 CALL pred ho5.main/2-0 (cc_multi) ho5.m:8
mdb> echo on
Command echo enabled.
mdb> register --quiet
mdb> break p
 0: + stop  interface pred ho5.p/2-0 (det)
mdb> continue
       3:      3  3 CALL pred ho5.p/2-0 (det) ho5.m:18 (exception.m:NNNN)
mdb> finish
      12:      3  3 EXCP pred ho5.p/2-0 (det) c2; ho5.m:18 (exception.m:NNNN)
mdb> dd
Call p(1, _)
Throws zero
Expected? no
q(1, 0)
Valid? yes
Call r(0, _)
Throws zero
Expected? yes
Found unhandled exception:
p(1, _)
zero
Is this a bug? yes
      12:      3  3 EXCP pred ho5.p/2-0 (det) c2; ho5.m:18 (exception.m:NNNN)
mdb> continue
exception(univ_cons('<<function>>'))
     401:    175  3 CALL pred ho5.p/2-0 (det) ho5.m:18 (exception.m:NNNN)
mdb> finish
     410:    175  3 EXCP pred ho5.p/2-0 (det) c2; ho5.m:18 (exception.m:NNNN)
mdb> dd
Call p(2, _)
Throws zero
Expected? no
q(2, 0)
Valid? yes
Found unhandled exception:
p(2, _)
zero
Is this a bug? yes
     410:    175  3 EXCP pred ho5.p/2-0 (det) c2; ho5.m:18 (exception.m:NNNN)
mdb> continue
exception(univ_cons('<<function>>'))
